.gitignore

```diff
@@ -37,6 +37,8 @@ Temporary Items
 /lib/cache/
 /lib/store/
 /lib/target/
+/lib/cache/
+/lib/store/
```

Comment:
These are already on main. Adding them here is creating duplicates.

Reply:
Removed